config/routes.rb

```diff
@@ -12,8 +12,15 @@
   get '/forks', to: 'insights#forks'
   get '/signup', to: 'users#new'
   post '/signup', to: 'users#create'
+  get '/profile', to: 'users#profile'
   get '/login', to:'sessions#new'
   post '/login', to:'sessions#create'
   delete '/logout', to:'sessions#destroy'
+  get '/repository/new', to: 'repository#new'
+  get '/repository', to: 'repository#show'
+  get 'repository/create'
+  get 'repository/edit'
+  get 'repository/destroy'
```

Comment:
Do we use get 'repository/create', get 'repository/edit', get 'repository/destroy'?